src/main/java/teammates/ui/webapi/output/FeedbackQuestionData.java

```diff
@@ -176,4 +180,18 @@ public Integer getCustomNumberOfEntitiesToGiveFeedbackTo() {
     public List<FeedbackVisibilityType> getShowRecipientNameTo() {
         return showRecipientNameTo;
     }
+
+    /**
+     * Hides weights in question details if present.
```

Comment:
Again, this is implementation details. Javadoc header comment shouldn't mention specifics.